sdk/storage/Azure.Storage.Queues/src/QueueClientConfiguration.cs

```diff
@@ -15,18 +16,22 @@ internal class QueueClientConfiguration : StorageClientConfiguration
 
         public QueueMessageEncoding MessageEncoding { get; internal set; }
 
+        public SyncAsyncEventHandler<InvalidMessageEventArgs> InvalidMessageHandler { get; internal set; }
```

Comment:
Super nit - consider calling these `Handlers` so readers are reminded it's a multicast delegate.